<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WTextSourceViewerSearchUI</class>
 <widget class="QWidget" name="WTextSourceViewerSearchUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>45</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="search_text_p"/>
   </item>
   <item>
    <widget class="QToolButton" name="previous_p">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="coveragebrowser.qrc">
       <normaloff>:/images/previous.svg</normaloff>:/images/previous.svg</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="next_p">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="coveragebrowser.qrc">
       <normaloff>:/images/next.svg</normaloff>:/images/next.svg</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="case_sensitive_p">
     <property name="text">
      <string>Case sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="whole_word_p">
     <property name="text">
      <string>Whole word</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="close_p">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="coveragebrowser.qrc">
       <normaloff>:/images/close.svg</normaloff>:/images/close.svg</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="coveragebrowser.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close_p</sender>
   <signal>clicked()</signal>
   <receiver>search_text_p</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
